src/styles/main.scss

```diff
@@ -0,0 +1,52 @@
+.stopwatch {
+  width: 80vmin;
+  height: 80vmin;
+  position: relative;
+  border: 2px solid black;
+  border-radius: 50%;
+  margin: 0 auto;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.minute-hand,
+.second-hand {
+  background-color: black;
+  position: absolute;
+  left: calc(50% - 2px);
+  bottom: calc(50% + 5px);
+  transform-origin: bottom center;
+  transform: rotate(0deg);
+  transition: transform 5s linear;
+}
+
+.second-hand {
+  width: 2px;
+  height: 35vmin;
+  animation: rotate-seconds 60s linear infinite;
+}
+
+.minute-hand {
+  width: 4px;
+  height: 30vmin;
+  animation: rotate-minutes 3600s linear infinite;
```

Comment:
The animation duration for the minute hand should be 60s for the demo purpose, not 3600s.

Reply:
I'm sorry! I forgot to fix the arrow